br/pkg/utils/error_handling.go

```diff
@@ -0,0 +1,181 @@
+package utils
```

Comment:
new file should add license first, you can copy from other files.

Reply:
ah! good catch